src/index.js

```diff
@@ -40,6 +42,8 @@ itemDescription.addEventListener('keypress', (e) => {
 
     // Update index of added Element
     UpdateInformation.updateIndex();
```

Comment:
- [ ] [OPTIONAL] The list should start from 1, not 0, you can fix this so you save time in the next milestone
![Screenshot_20220604_084305](https://user-images.githubusercontent.com/25227022/171989933-aee67154-4181-437c-a8a1-0d58e868cbda.png)